docs/docs/code/nodejs/auth/README.md

```diff
@@ -6,58 +6,66 @@ This document applies only to [Node.js workflow code steps](/workflows/steps/cod
 
 When you use [prebuilt actions](/components/actions/) tied to apps, you don't need to write the code to authorize API requests. Just [connect your account](/connected-accounts/#connecting-accounts) for that app and run your workflow.
 
-But sometimes you'll need to [write your own code](/workflows/steps/code/). You can also connect apps to custom code steps, using the auth information to authorize requests to that app.
+But sometimes you'll need to [write your own code](/code/nodejs). You can also connect apps to custom code steps, using the auth information to authorize requests to that app.
 
-For example, you may want to send a Slack message from a step. We use Slack's OAuth integration to authorize requests to Slack, so you can connect your account to this step and send a message using the access token Pipedream generates for you at `auths.slack.oauth_access_token`:
+For example, you may want to send a Slack message from a step. We use Slack's OAuth integration to authorize requests to Slack, so you can connect your account to this step and send a message using the access token Pipedream generates for you at `$.auths.slack.oauth_access_token`:
```

Comment:
Like `$.service.db`, auth data is exposed to components by adding the associated app prop (just like in normal components). So you'll want to rewrite these sections accordingly.

Then you can reference auth data via `this.slack.oauth_access_token` (for example), not `$.auths`.

Reply:
Can you double check that?

I've tried in my workflow and API keys are only available under `this.slack.$auths`.

```js
export default defineComponent({
  props: {
    slack: {
      type: 'app',
      app: 'slack'
    }
  },
  async run({ steps, $ }) {
    console.log(this.slack.oauth_access_token);
  }
});

```

Reply:
`this.appName.$auths` is the way currently, closing out 👍